proposals/lip-0053.md

```diff
@@ -105,7 +105,7 @@ The Interoperability module supports two commands used to certify the state of a
 
 ### Notation and Constants
 
-All interoperability constants are defined in [LIP 0045][lip-0045#constants].
+All interoperability constants are defined in [LIP 0045][lip-0045#constants]. `SMTVerifyInclusionProof` is the `verify_inclusion_proof` function specified in [LIP 0039][lip-0039].
```

Comment:
We have always used `camelCase` for functions in LIPs, same as in the Lisk SDK. I don't understand why we deviate for it here and in LIP 39. Moreover, it would be nice to just use the same function name here and in LIP 39.

Reply:
I think it's better to be specific on the verify function used as in this LIP we also have some functions related to the regular Merkle tree. We could rename the functions in LIP0039 directly, otherwise